homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -196,6 +197,11 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
         },
         MiroboVacuum.async_goto.__name__,
     )
+    platform.async_register_entity_service(
+        SERVICE_CLEAN_SEGMENT,
+        {vol.Required("segments"): vol.Any(vol.Coerce(int), [vol.Coerce(int)])},
```

Comment:
We should use `cv.ensure_list` instead to make sure segments is a list of integers.